src/context/src/input.rs

```diff
@@ -31,4 +38,14 @@ impl InputHandler {
     pub fn key_down(&self, key: VirtualKeyCode) -> bool {
         self.keys_down.contains(&key)
     }
-}
\ No newline at end of file
+
+    ///Check if `button` is pressed.
+    pub fn button_down(&self, button: MouseButton) -> bool {
```

Comment:
Should this be called `button_down`? It is obvious from the type `button` has, but I feel like button_down is a suboptimal name.